derive/src/parse.rs

```diff
@@ -178,10 +182,11 @@ pub fn next_group(source: &mut Peekable<impl Iterator<Item = TokenTree>>) -> Opt
     }
 }
 
-#[allow(dead_code)]
```

Comment:
I am totally fine to comment out/remove `debug_current_token`

It is useful during development, but it doesn't worth an effort of making it no_std compatible

Reply:
Commented it out.